### PR TITLE
Feat wallet

### PR DESCRIPTION
**Pull request by JakubKostrzewsky**
## Description

Get wallet projects

## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: test/features/project/presentation/wallet/bloc/wallet_cubit_test.dart [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+    blocTest<WalletCubit, WalletState>(
+      'emits [success] when user get list of projects successful',
+      build: () {
+        when(() => getWalletProjectsUseCase()).thenAnswer((_) async => Right(mockedProjects));
+
+        return WalletCubit(getWalletProjectsUseCase);
+      },
+      act: (WalletCubit bloc) => bloc.load(),
+      expect: () => <WalletState>[
```
Comment (mhnowak) [inline test/features/project/presentation/wallet/bloc/wallet_cubit_test.dart:30]: no `const`?

File: test/features/project/presentation/wallet/bloc/wallet_cubit_test.dart [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    blocTest<WalletCubit, WalletState>(
+      'emits [failure] when user get list of projects unsuccessful',
+      build: () {
+        when(() => getWalletProjectsUseCase()).thenAnswer((_) async => const Left(failure));
+
+        return WalletCubit(getWalletProjectsUseCase);
+      },
+      act: (WalletCubit bloc) => bloc.load(),
+      expect: () => <WalletState>[
+        const WalletState.failed(failure),
```
Comment (mhnowak) [inline test/features/project/presentation/wallet/bloc/wallet_cubit_test.dart:48]: move `const` before list declaration

File: test/helpers/mocked_data.dart [hunk 3]
```diff
@@ -13,18 +13,29 @@ final List<DonationHistoryEntity> mockedDonationHistory = [
...
-          totalAmount: 1000,
-        ),
-      ],
+      donators: [mockedDonator],
       budget: 3000,
     ),
   ),
 ];
 
+final DonatorEntity mockedDonator = DonatorEntity(
```
Comment (mhnowak) [inline test/helpers/mocked_data.dart:22]: why not const?

File: test/helpers/mocked_data.dart [hunk 4]
```diff
@@ -13,18 +13,29 @@ final List<DonationHistoryEntity> mockedDonationHistory = [
...
+  name: 'name',
+  pictureUrl: 'imageUrl',
+  totalAmount: 100,
+);
+
+final List<ProjectEntity> mockedProjects = [
```
Comment (mhnowak) [inline test/helpers/mocked_data.dart:28]: why not const?

**Review by mhnowak — CHANGES_REQUESTED** [review]

File: test/helpers/mocked_data.dart (hunk 4 above)
Comment (JakubKostrzewsky) [reply test/helpers/mocked_data.dart:28]: I forgot to make const constructors on all entity classes, so lint didn't catch it. Thank!

**Pull request merged**